alienfile

```diff
@@ -32,10 +48,9 @@ share {
         '%{make} install',
     ];
 
-};
+    after 'gather' => sub {
+        my($build) = @_;
+        $build->runtime_prop->{command} = 'tidy';
```

Comment:
For a share install the command should always be called `tidy`.  Unless the upstream developer decides to change the name.